src/main/java/de/tum/in/www1/artemis/web/rest/TextAssessmentResource.java

```diff
@@ -145,7 +145,7 @@ public ResponseEntity<Result> saveTextAssessment(@PathVariable Long participatio
     @PreAuthorize("hasRole('TA')")
     public ResponseEntity<Result> saveTextExampleAssessment(@PathVariable long exerciseId, @PathVariable long exampleSubmissionId, @RequestBody TextAssessmentDTO textAssessment) {
         log.debug("REST request to save text example assessment : {}", exampleSubmissionId);
-        Optional<ExampleSubmission> optionalExampleSubmission = exampleSubmissionRepository.findBySubmissionId(exampleSubmissionId);
+        Optional<ExampleSubmission> optionalExampleSubmission = exampleSubmissionRepository.findById(exampleSubmissionId);
```

Comment:
This change needs some testing. Instead of using the parameter as the example submission id (like documented), it used the value as the submission id. I checked the client and it looked like the example submission id was always passed, but we should double-check that and ensue saving an example submission text assessment still works.